src/prob/prob_init_fluid.cpp

```diff
@@ -210,7 +210,11 @@ void incflo::init_taylor_green3d (Box const& vbx, Box const& /*gbx*/,
     {
         Real x = Real(i+0.5)*dx[0];
         Real y = Real(j+0.5)*dx[1];
+#if (AMREX_SPACEDIM == 3)
```

Comment:
Can I ask why you chose to do this rather than only having the function for 3D? SInce this function is init_taylor_green3d, one might argue that it's only meant for 3d. But I suppose it doesn't really matter either way.